cli/internal/inference/inference.go

```diff
@@ -23,96 +23,96 @@ const (
 
 var _frameworks = []Framework{
 	{
-		Slug:       "blitzjs",
-		EnvMatcher: "^NEXT_PUBLIC_",
+		Slug:         "blitzjs",
+		EnvWildcards: []string{"NEXT_PUBLIC_*"},
```

Comment:
Nice

Reply:
Eat your own dog food! Also, enables us to get more-clever about these.